code/modules/mob/mob_grab_vr.dm

```diff
@@ -0,0 +1,14 @@
+//TFF 12/8/19 - Port ChompStation facesitting
+/obj/item/weapon/grab/handle_eye_mouth_covering(mob/living/carbon/target, mob/user, var/target_zone)
+	var/announce = (target_zone != last_hit_zone) //only display messages when switching between different target zones
+	last_hit_zone = target_zone
+
+	switch(target_zone)
+		if(BP_HEAD)
+			if(force_down)
+				if(announce)
+					assailant.visible_message("<span class='warning'>[assailant] moves their ass to [target]'s head, sitting down on them, making them unable to see anything else than [assailant]'s butt!</span>")
```

Comment:
This description does not to be as long-winded. A simple `[assailant] sits on [target]'s head` would be better.

Reply:
Already have that ready for my next commit but I've been working on trying to change this code so that the snippet in mob_grab.dm works as well as this one.